spanner/cloud-client/src/test/java/com/example/spanner/SpannerSampleIT.java

```diff
@@ -185,13 +182,13 @@ public void testSample() throws Exception {
     assertThat(out).contains("12 Melissa Garcia");
 
     runSample("writewithtransactionusingdml");
-    out = runSample("querymarketingbudget");    
-    assertThat(out).contains("1 1 1800000");
-    assertThat(out).contains("2 2 200000");
-    
+    out = runSample("querymarketingbudget");
+    assertThat(out).contains("1 1 200000");
```

Comment:
Once the update/fix noted in my comment above is made this value should be "2200000". 
Thus this assert statement should be updated to be:

assertThat(out).contains("1 1 2200000");